academic-services/knowledge-exploration-service/TOC.yml

```diff
@@ -15,8 +15,12 @@
 - name: How to
   expanded: true
   items:
-  - name: Deploy new MAKES releases to existing MAKES instances
-    href: how-to-deploy-new-indexes-to-makes.md
+  - name: Create Index from MAG
+    href: how-to-create-index-from-mag.md
+  - name: Integrate with Existing Search
+    href: how-to-integrate-with-existing-search.md
```

Comment:
I thought we weren't shipping with either of these?

Reply:
We aren't, I'm guessing Alvin needs to merge w/ v2-launch.  (or this is a bad merge?)

---
In reply to: [376622871](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/105#discussion_r376622871) [](ancestors = 376622871)